### DISCUSSION TITLE
Trying to use the what’s app bot general ChatBot template, says there is an issue

### DISCUSSION THREAD

**Discussion by earthprimeone** [AntlerVC/firetable#1360] [Support / Q&A] [answered]
It just won’t work

**Comment by harinij** [comment]
Hi @earthprimeone, the few places to check are the whatsapp phone number format and twilio config. We are working on a new iteration of the whatsapp bot that will make it more simpler and easier to use. We will keep you posted.

**Comment by earthprimeone** [comment]
Is there an eta on that at all?

On Fri, Jul 28, 2023 at 11:59 PM Harini Janakiraman <
***@***.***> wrote:

> Hi @earthprimeone <https://github.com/earthprimeone>, the few places to
> check are the whatsapp phone number format and twilio config. We are
> working on a new iteration of the whatsapp bot that will make it more
> simpler and easier to use. We will keep you posted.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rowyio/rowy/discussions/1360#discussioncomment-6581199>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7D3HZWMYSXWOTMP4BDRNJTXSSRDHANCNFSM6AAAAAA24HRXSU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
[image: photo]
Hafiz Ladhani
Chairman, HafizLadhani.com
+15879743506
| www.linkr.com/chairmanhafiz  <http://www.linkr.com/chairmanhafiz>
***@***.***  ***@***.***>
[image: facebook] <https://facebook.com/rightenergy> [image: instagram]
<https://instagram.com/chairmanhafiz> [image: linkedin]
<https://ca.linkedin.com/in/anifamilyoffice> [image: twitter]
<https://twitter.com/chairmanhafiz> [image: youtube]
<https://www.youtube.com/channel/UCFfV-uHo9YBxKwrq1ILhAtA> [image:
pinterest] <https://www.pinterest.com/chairmanhafiz>

Create your own email signature
<https://www.wisestamp.com/create-own-email-signature/?utm_source=promotion&utm_medium=signature&utm_campaign=create_your_own&srcid=>

**Comment by earthprimeone** [comment]
Hello is there any progress on the new iteration of the app? I would like
to try installing it as soon as possible, last time I had a few interface
issues with the setup and couldn't enter in info on either pc or iPad.
Looking forward to hearing from you

**Comment by earthprimeone** [comment]
Was able to run thru the install of whatsapp bot template using ipad,
however once the wizard completed i am now unsure what to do next to use /
test the app. Any advice?

**Reply by harinij** [reply, marked as answer]
Hi @earthprimeone the new iteration of it is now ready to use. Can you email me at harini@rowy.io to get access? This will be super easy to use visually.
